hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/SPUIDefinitions.java

```diff
@@ -347,6 +347,7 @@
      * Sort order of column - created at in target table.
      */
     public static final Direction TARGET_TABLE_CREATE_AT_SORT_ORDER = Direction.ASC;
+    public static final Direction TARGET_TABLE_LASTMODIFIED_AT_SORT_ORDER = Direction.DESC;
```

Comment:
Please check if `TARGET_TABLE_CREATE_AT_SORT_ORDER` can be removed

Reply:
Thank you for the review comment!
The changes have been committed. 
Please review.